change/@fluentui-eslint-plugin-12ef753c-74ec-4c3c-8e65-e5121c05bde3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=28887&commitId=48ba0249320d6b1e9a0108bbceaf5f525096a92b&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Root - RTL.default.chromium.png|646| Changed|
|MessageBar.Root dismiss.default.chromium.png|718| Changed|
|MessageBar.Root dismiss - RTL.default.chromium.png|718| Changed|
|MessageBar.Root.default.chromium.png|646| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>